packages/azure-storage/files/helm-chart/templates/NFS.yaml

```
@@ -0,0 +1,47 @@
+---
+apiVersion: storage.azure.upbound.io/v1beta1
+kind: Account
+metadata:
+  labels:
+    app.kubernetes.io/name: {{ .Chart.Name }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/part-of: {{ .Values.organization }}.{{ .Values.project }}
+    app.kubernetes.io/managed-by: kubepak
+  name: {{ .Values.organization }}{{ .Values.project }}nfs
+spec:
+  forProvider:
+    accountKind: "BlockBlobStorage"
+    accountTier: "Premium"
+    accountReplicationType: ZRS
+    allowNestedItemsToBePublic: false
+    infrastructureEncryptionEnabled: true
+    isHnsEnabled: true
```

Do we need all the time _hierarchical namespace_?
```suggestion
    isHnsEnabled: "{{ index .Values "packages" "azure-storage" "nfs" "hnsEnabled"}}"
```